lib/puppet/face/ca.rb

```diff
@@ -168,8 +168,8 @@
 
   action :sign do
     summary _("Sign an outstanding certificate request.")
-    option("--[no-]allow-dns-alt-names") do
-      summary _("Whether or not to accept DNS alt names in the certificate request")
+    option("--[no-]allow-subject-alt-names") do
+      summary _("Whether or not to accept alt names in the certificate request")
```

Comment:
This one should look like the one for the `cert` command up above: both options specified, pointing to the same hash key, `dns` with a deprecation warning.